app/components/markdown/index.js

```diff
@@ -3,16 +3,18 @@
 
 import {connect} from 'react-redux';
 
-import {getAutolinkedUrlSchemes} from 'mattermost-redux/selectors/entities/general';
+import {getAutolinkedUrlSchemes, getConfig} from 'mattermost-redux/selectors/entities/general';
 import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
 import {getCurrentUserMentionKeys} from 'mattermost-redux/selectors/entities/users';
 
 import Markdown from './markdown';
 
 function mapStateToProps(state) {
+    const config = getConfig(state);
     return {
         autolinkedUrlSchemes: getAutolinkedUrlSchemes(state),
         mentionKeys: getCurrentUserMentionKeys(state),
+        minimumHashtagLength: parseInt(config.MinimumHashtagLength, 10),
```

Comment:
what is the default value for this? also this config value does not exists in previous server versions and the app should be backwards compatible, can you make sure to check the existence of the config property before assigning it.

Reply:
Thanks @enahum!
I think it had better to set 3 as default value, because 3 is previous minimumHashtagLength. I've fix this, and I made sure that it works fine for v5.6 server.